aws/resource_aws_route53_record_test.go

```diff
@@ -248,106 +246,141 @@ func TestAccAWSRoute53Record_basic_fqdn(t *testing.T) {
 
 func TestAccAWSRoute53Record_txtSupport(t *testing.T) {
 	var record1 route53.ResourceRecordSet
+	resourceName := "aws_route53_record.default"
 
 	resource.ParallelTest(t, resource.TestCase{
 		PreCheck:        func() { testAccPreCheck(t) },
-		IDRefreshName:   "aws_route53_record.default",
+		IDRefreshName:   resourceName,
 		IDRefreshIgnore: []string{"zone_id"}, // just for this test
 		Providers:       testAccProviders,
 		CheckDestroy:    testAccCheckRoute53RecordDestroy,
 		Steps: []resource.TestStep{
 			{
 				Config: testAccRoute53RecordConfigTXT,
 				Check: resource.ComposeTestCheckFunc(
-					testAccCheckRoute53RecordExists("aws_route53_record.default", &record1),
+					testAccCheckRoute53RecordExists(resourceName, &record1),
 				),
 			},
+			{
+				ResourceName:            resourceName,
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"allow_overwrite", "weight", "zone_id"},
```

Comment:
Should `zone_id` be included here?

```suggestion
				ImportStateVerifyIgnore: []string{"allow_overwrite", "weight"},
```

Reply:
According to my notes, it needs to go into my soon-to-be-created `technical-debt` ticket to fix imports that are currently sad. If you'd prefer, I can add `#TODO` comments for each of those in the code as well - let me know what you think!

Reply:
Having them documented in #10002 is good for now. 👍